File: Clases 2018/Clases-C#/Simulacro_parcial_1_2018/_Resolucion/C#_RegistroUsuarios/RegistroUsuarios/frmDetalleUsuario.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace RegistroUsuarios.GUILayer
{
    public partial class frmDetalleUsuario : Form
    {
        private string idUsuario = string.Empty;
        
        public frmDetalleUsuario(string id)
        {
            InitializeComponent();
            idUsuario = id;
        }

        private void frmDetalleUsuario_Load(object sender, EventArgs e)
        {
            DataTable tabla = BDHelper.getBDHelper().ConsultaSQL("SELECT id_perfil, n_perfil FROM Perfiles");
            GUIHelper.getGUIHelper().llenarCombo(cboPerfil, tabla, "n_perfil", "id_perfil");
            string consulta = "SELECT * FROM Usuarios WHERE id_usuario = " + idUsuario;
            DataTable usuario = BDHelper.getBDHelper().ConsultaSQL(consulta);
            txtNombre.Text = usuario.Rows[0].ItemArray[1].ToString();
            txtCorreoElectronico.Text = usuario.Rows[0].ItemArray[3].ToString();
            txtPassword.Text = usuario.Rows[0].ItemArray[2].ToString();
            txtConfirmarPassword.Text = usuario.Rows[0].ItemArray[2].ToString();
            cboPerfil.SelectedValue = usuario.Rows[0].ItemArray[4].ToString();
        }

        private void btnVolver_Click(object sender, EventArgs e)
        {
            this.Close();

        }


    }
}

File: Clases 2018/Clases-C#/Simulacro_parcial_1_2018/_Resolucion/VB_RegistroUsuarios/RegistroUsuarios/frmDetalleUsuario.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Public Class frmDetalleUsuario
    Inherits Form

    Private idUsuario As String = String.Empty

    Public Sub New(ByVal id As String)
        InitializeComponent()
        idUsuario = id
    End Sub

    Private Sub frmDetalleUsuario_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
        Dim tabla As DataTable = BDHelper.getBDHelper().ConsultaSQL("SELECT id_perfil, n_perfil FROM Perfiles")
        GUIHelper.getGUIHelper().llenarCombo(cboPerfil, tabla, "n_perfil", "id_perfil")
        Dim consulta As String = "SELECT * FROM Usuarios WHERE id_usuario = " & idUsuario
        Dim usuario As DataTable = BDHelper.getBDHelper().ConsultaSQL(consulta)
        txtNombre.Text = usuario.Rows(0).ItemArray(1).ToString()
        txtCorreoElectronico.Text = usuario.Rows(0).ItemArray(3).ToString()
        txtPassword.Text = usuario.Rows(0).ItemArray(2).ToString()
        txtConfirmarPassword.Text = usuario.Rows(0).ItemArray(2).ToString()
        cboPerfil.SelectedValue = usuario.Rows(0).ItemArray(4).ToString()
    End Sub

    Private Sub btnVolver_Click(sender As Object, e As EventArgs) Handles btnVolver.Click
        Me.Close()
    End Sub
End Class

